lms/static/sass/inline-discussion-rtl.scss

```diff
@@ -0,0 +1,34 @@
+// From lms-main-v1-rtl.scss
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-rtl'; // set the layout for right to left languages
+@import 'base/variables-rtl';
```

Comment:
@itsjeyd ok - looks like there are two ways to properly handle this situation than:

1. RTL file should only contain direction var and should never clash with base var file. Inthis case it might be good to add a comment to *-rtl.* file(s).
2. RTL file is allowed to contain any styles and it should handle style overrides properly; in this case importing RTL before base is a bug that haven't exposed itself yet because RTL file does not contain any overrides.

Anyway, both are totally unrelated to this PR, so we can handle it separately, or just let it be until it becomes an actual bug (or is fixed occasionally in some future PR).